### DISCUSSION TITLE
why the replacement of vim %s does not work?

### DISCUSSION THREAD

**Discussion by GaryLiuTelus** [VSCodeVim/Vim#7805] [Q&A]
![vim_replace](https://user-images.githubusercontent.com/39339422/176515579-4b0e35c2-f2f8-4ee3-aacb-ac8b080bbbb6.gif)
Anyone can help me figure out why the replace of vscode vim does not work when obviously there are matches, it said pattern not found?

**Comment by J-Fields** [comment]
Hmm, this works for me. Can you confirm you're on the latest version (1.23.1)?

**Comment by GaryLiuTelus** [comment]
Yes, I am using the latest 1.23.1. I tried to re-start vscode, or disabled all other extensions I am using, but the issue persisted. I have to use vscode replace instead.

**Reply by J-Fields** [reply]
Would you mind providing your settings (and `.vimrc`, if you're using it)?

**Comment by GaryLiuTelus** [comment]
I am not using vimrc setting in vscode

![image](https://user-images.githubusercontent.com/39339422/176735846-a8286476-d27e-40bd-806f-1b63e8a0e7aa.png)

**Reply by J-Fields** [reply]
Do you have any other `vim.*` settings changed from their default? What operating system are you using?

**Reply by GaryLiuTelus** [reply]
Sorry, I was on vacation and missed your reply. 
I am using Windows 10 20H2 19042.1766. After upgraded to the latest VS Code 1.69, the problem is still there. 
I don't think I have any other vim.* settings changed, or do you have any suggestions (files or settings) that I need to check? 

Thanks!

**Reply by GaryLiuTelus** [reply]
Hi @J-Fields ,

I found the reason. 

I installed neovim extension, and by default it enabled the option: Use neovim to execute Ex command. And then I uninstalled neovim extension, but the option still got checked. After unchecked the option, the replace is now working. 

Thanks for all your help!
